docs/releases/v8_20.md

```diff
@@ -0,0 +1,39 @@
+# Release Notes
+
+## July 2023, vol 2 (version 8.20)
+
+### Overview
+
+The **July 29th, 2023** release of **DietPi v8.20** comes with a new software title `Homebridge` and a couple of improvements and bug fixes.
+
+![Netherland north sea](../assets/images/dietpi-release-v8_20.jpg){: width="480" height="320" loading="lazy"}
```

Comment:
```suggestion
![Netherlands North Sea](../assets/images/dietpi-release-v8_20.jpg){: width="480" height="320" loading="lazy"}
```
Since "North Sea" is a name 🙂.